src/sinks/appsignal/mod.rs

```diff
@@ -0,0 +1,265 @@
+#[cfg(all(test, feature = "appsignal-integration-tests"))]
```

Comment:
Would be great to see a module level doc comment to describe briefly what the integration is for and how it works.

We do this in some other sinks, grep for `//!` . For example `loki`, `datadog_logs` , `amqp` (for a minimal example)